packages/block-editor/src/components/writing-flow/index.js

```diff
@@ -44,7 +44,7 @@ export function useWritingFlow() {
 			useArrowNav(),
 			useRefEffect(
 				( node ) => {
-					node.tabIndex = -1;
+					node.tabIndex = 0;
```

Comment:
Hmm, I wonder if this adds an empty tab stop. Not real familiar with how this code works, but doesn't this refer to the virtual DOM div that gets an adjusted tabindex? Maybe this is just the initial value or something.

Reply:
This is on the writing flow wrapper, so it shouldn't add an extra tab stop because we intercept tab before and after the writing flow container. Alternatively, I think we can set set this tab index only when multiple blocks are selected (because this is where we move focus), but I don't think it hurts to add it permanently.